openreview/venue/venue.py

```diff
@@ -1053,6 +1061,244 @@ def set_SAC_ethics_review_process(self, sac_ethics_flag_duedate=None):
     def open_reviewer_recommendation_stage(self, start_date=None, due_date=None, total_recommendations=7):
         self.invitation_builder.set_reviewer_recommendation_invitation(start_date, due_date, total_recommendations)
 
+    def ithenticate_create_and_upload_submission(self):
+        if not self.iThenticate_plagiarism_check:
+            raise openreview.OpenReviewException(
+                "iThenticatePlagiarismCheck is not enabled for this venue."
+            )
+
+        self.invitation_builder.set_iThenticate_plagiarism_check_invitation()
+
+        iThenticate_client = openreview.api.iThenticateClient(
+            self.iThenticate_plagiarism_check_api_key,
+            self.iThenticate_plagiarism_check_api_base_url,
+        )
+
+        edges = self.client.get_grouped_edges(invitation=self.get_iThenticate_plagiarism_check_invitation_id(), groupby='head')
+        edges_dict = {edge["id"]["head"]: edge["values"] for edge in edges}
+
+        submissions = self.get_submissions()
+        for submission in tqdm(submissions):
+            # TODO - Decide what should go in metadata.group_context.owners
```

Comment:
what should we do with this TODO? do we need to send this data?

Reply:
I believe this is the group's owners (which should be the PCs in this case?). We successfully made requests without it so we could leave it out as well.